api/src/main/resources/workbench.yaml

```diff
@@ -2159,10 +2175,58 @@ paths:
           schema:
             $ref: "#/definitions/DataSetPreviewResponse"
 
+  /v1/offline/syncResearcherData:
```

Comment:
[API design thought] I know we sort of had a pattern with "offline" in the URL path, but in this context it feels a little awkward. It might fit better with two changes:

1. Replace "offline" with "rdrExport" here, so the URL path here would become /v1/rdrExport/syncResearcherData
2. Remove "offline" from the controller, so it becomes RdrExportController.